.github/workflows/test.yml

```
@@ -107,68 +107,72 @@ jobs:
           test -f debian/artifacts/test_1_amd64.buildinfo
           test -f debian/artifacts/test_1_amd64.changes
 
-  local-dockerfile:
+  extra-docker-args:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - run: cat test/Makefile_local-dockerfile >> test/Makefile
+      - run: cat test/Makefile_extra-docker-args >> test/Makefile
       - uses: ./
-        env:
+        env: 
```

To make it perfect: There is a lost space :wink: 
```suggestion
        env:
```

---

Oops, must have been auto formatted :smiley: